cmd/helm-operator/main.go

```diff
@@ -116,7 +120,14 @@ func main() {
 
 	// init go-kit log
 	{
-		logger = log.NewLogfmtLogger(os.Stderr)
+		switch *logFormat {
+		case "json":
+			logger = log.NewJSONLogger(log.NewSyncWriter(os.Stderr))
+		case "fmt":
+			logger = log.NewJSONLogger(log.NewSyncWriter(os.Stderr))
```

Comment:
I think this should be

```suggestion
			logger = log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
```